src/sdk/modules/validators/ownableValidator.test.ts

```diff
@@ -0,0 +1,228 @@
+import {
+  getAddOwnableValidatorOwnerAction,
+  getOwnableValidator,
+  getOwnableValidatorSignature,
+  getSetOwnableValidatorThresholdAction
+} from "@rhinestone/module-sdk"
+import {
+  http,
+  type Account,
+  type Address,
+  type Chain,
+  type PublicClient,
+  encodePacked,
+  encodeAbiParameters,
+  encodeFunctionData,
+  parseAbi,
+  publicActions,
+  zeroAddress
+} from "viem"
+import { afterAll, beforeAll, describe, expect, test } from "vitest"
+import { TEST_CONTRACTS } from "../../../test/callDatas"
+import { toNetwork } from "../../../test/testSetup"
+import {
+  fundAndDeployClients,
+  getBalance,
+  getTestAccount,
+  getTestSmartAccountClient,
+  killNetwork,
+  toTestClient
+} from "../../../test/testUtils"
+import type { MasterClient, NetworkConfig } from "../../../test/testUtils"
+import addresses from "../../__contracts/addresses"
+import {
+  type NexusClient,
+  createNexusClient
+} from "../../clients/createNexusClient"
+import { createOwnableValidatorModule } from "../.."
+
+describe("modules.ownableValidator", async () => {
+  let network: NetworkConfig
+  let chain: Chain
+  let bundlerUrl: string
+
+  // Test utils
+  let testClient: MasterClient
+  let account: Account
+  let nexusClient: NexusClient
+  let nexusAccountAddress: Address
+  let recipient: Account
+  let recipientAddress: Address
+
+  beforeAll(async () => {
+    network = await toNetwork("FILE_LOCALHOST")
```

Comment:
Just FYI this is the default setting now. `await toNetwork()` works the same way